login-page/loginPage.css

```diff
@@ -120,19 +113,20 @@ a{
     font-size: 1.5rem;
     color: var(--facebook-blue);
     font-weight: 540;
+    margin-top: 1.60rem;
 }
 
 .login-forgot-password{
     font-size: 1.25rem;
     color: var(--facebook-blue);
-    margin-top: 0.3rem;
+    margin-top: 1.82rem;
 }
 
 .signup{
     background: white;
     border: 0.1rem solid var(--mild-grey);
     padding: 2rem 3rem;
-    width: 29rem;
+    width: var(--container-width);
 }
 
 .signup-text{
```

Comment:
Please remove margin from this class as it is not required

Reply:
taking default margins by default which is larger than 0.48 rem.